src/app/organizations/organization-members/organization-members.component.html

```diff
@@ -38,17 +38,20 @@
             <div fxLayout="row" fxLayoutAlign="space-between stretch">
               <div fxFlex fxLayout="row" fxLayoutGap="1rem" fxLayoutAlign="center center">
                 <img
-                  *ngIf="organizationUser.user?.avatarUrl"
+                  *ngIf="organizationUser.user?.avatarUrl; else noAvatar"
                   mat-card-avatar
                   class="header-image"
                   [src]="organizationUser.user?.avatarUrl"
+                  alt="User avatar"
                 />
-                <img
-                  *ngIf="!organizationUser.user?.avatarUrl"
-                  mat-card-avatar
-                  class="header-image"
-                  src="https://via.placeholder.com/300x300?text=Avatar+Not+Found"
-                />
+                <ng-template #noAvatar>
+                  <img
+                    mat-card-avatar
+                    class="header-image"
+                    src="https://via.placeholder.com/300x300?text=Avatar+Not+Found"
```

Comment:
Was like this before, but https://placeholder.com/ doesn't seem like an obvious source. 
https://www.reddit.com/r/web_design/comments/10lqgja/placeholdercom_is_no_more/ looks like it got bought up which means the image could be replaced unexpectedly. 

Can you find an alternative and/or look for other sources? (also see https://via.placeholder.com/150 elsewhere)